src/app/exams/exams-preview.component.ts

```diff
@@ -1,9 +1,9 @@
 import { Component, Inject, ViewChild } from '@angular/core';
-import { MAT_DIALOG_DATA } from '@angular/material';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material';
 
 @Component({
   template: `
-    <planet-exams-view [isDialog]="true" [questionNum]="1" [exam]="data.exam"></planet-exams-view>
+    <planet-exams-view [isDialog]="true" [questionNum]="1" [exam]="data.exam" (closePreview)="closeDialog($event)"></planet-exams-view>
```

Comment:
@AlekseiKorolev You need to remove `$event` for this to build correctly.  The `closeDialog` method does not take any arguments.